src/UserSettingsPage.cpp

```diff
@@ -725,6 +725,7 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
         // TODO: Is there a way to limit to just emojis, rather than
         // all emoji fonts?
         auto emojiFamilies = fontDb.families(QFontDatabase::Symbol);
+        emojiFontSelectionCombo_->addItem(QString("default"));
```

Comment:
Selecting this should be handled specially, so that the font is not actually set to default. It should also be translatable.

Reply:
The reason I set it to "default" is because here https://github.com/Nheko-Reborn/nheko/blob/4aefac08a4b57e6fb344cd8cffb0b6e32698e0ba/src/UserSettingsPage.cpp#L104  . So it works properly without needing to specially handle it.

But yeah translations, so can I rewrite that to 
`emojiFontSelectionCombo_->addItem(tr("default"));`

and then
`emojiFont_         = settings.value("user/emoji_font_family", tr("default")).toString();`

I guess that would work right?

Reply:
Yeah, that should work. You may also want to do the same for normal fonts. I forgot we already handled default specifically.